src/psc/templates/homepage.jinja2

```diff
@@ -0,0 +1,153 @@
+{% extends "layout.jinja2" %}
+{% block main %}
+    <section class="hero p-1 is-warning">
+        <div class="hero-body">
+            <div class="columns">
+                <div class="column is-two-thirds m-6">
+                    <p class="title">
+                        PyScript Collective
+                    </p>
+                    <p class="subtitle">
+                        Software and people, from and for the PyScript Community.
+                    </p>
+                </div>
+                <div class="column is-2">
+                    <a href="{{ root_path }}/gallery/examples/hello_world/index.html">
+                        <div class="card">
+                            <div class="card-image">
+                                <figure class="image is-square">
+                                    <img alt="placeholder" src="{{ root_path }}/gallery/examples/hello_world/screenshot.png">
+                                </figure>
+                            </div>
+                            <div class="card-content">
+                                <div class="content">
+                                    Toroid Example
```

Comment:
Should this the `hello world` instead?

Reply:
Yes those were just placeholders to think about how it looks with a bunch of examples.